Extend docs related to Multicall (#785)
software-mansion/starknet.py: docs/guide/deploying_contracts.rst

Simple deploy
-------------

> If you already know the class hash of a contract you want to deploy (and the class hash is present on-chain, i.e. has been previously declared) just use the :meth:`Contract.deploy_contract`.

Reviewer: ```suggestion
If you already know the class hash of an already declared contract you want to deploy just use the :meth:`Contract.deploy_contract`.
```
